external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/EventHubSpout.java

```diff
@@ -121,8 +121,8 @@ public void preparePartitions(Map config, int totalTasks, int taskIndex, SpoutOu
         zkEndpointAddress = sb.toString();
       }
       stateStore = new ZookeeperStateStore(zkEndpointAddress,
-          (Integer)config.get(Config.STORM_ZOOKEEPER_RETRY_TIMES),
-          (Integer)config.get(Config.STORM_ZOOKEEPER_RETRY_INTERVAL));
+          Integer.parseInt(config.get(Config.STORM_ZOOKEEPER_RETRY_TIMES).toString()),
```

Comment:
It compiles :) 

The issue was that the values were being parsed as Long, and not Integer.
Instead of doing an explicit cast, I wanted to be safe and parse the numeric value.